kernel/Kconfig

```diff
@@ -1006,6 +1006,20 @@ config STATIC_INIT_GNU
 	  Note the option CMAKE_LINKER_GENERATOR does not yet support this feature
 	  or CPP.
 
+config BOOTARGS
+	bool "Support bootargs"
+	default n
```

Comment:
This line should be removed, since the default for boolean options is already n

Reply:
Done